docs/quickstart.rst

```diff
@@ -20,32 +70,90 @@ You can clone the repository and use `setuptools <https://github.com/pypa/setupt
 
 .. code-block:: bash
 
-    git clone https://github.com/ApeWorX/ape.git
-    cd ape
-    python3 setup.py install
+    $ git clone https://github.com/ApeWorX/ape.git
+    $ cd ape
+    $ python3 setup.py install
+
+via ``docker``
+==============
+
+Please visit our `Dockerhub <https://hub.docker.com/repository/docker/apeworx/ape>`_ for more details on using Ape with Docker.
+
+.. code-block:: bash
+
+    $ docker run \
+    --volume $HOME/.ape:/root/.ape \
+    --volume $HOME/.vvm:/root/.vvm \
+    --volume $HOME/.solcx:/root/.solcx \
+    --volume $PWD:/root/project \
+    --workdir /root/project \
+    apeworx/ape compile
+
+
+Troubleshooting: Python 3.6.X will have issues with installation with setup.py (blake2.py) Intel Processor x86
```

Comment:
Since we are not supporting python 3.6, it might be weird to include troubleshooting for it.

Reply:
https://github.com/ApeWorX/ape/pull/193/commits/589392e515e6a90d051884fbbb717a32f85011a2